system/mp/mpcomm.cpp

```diff
@@ -2567,11 +2570,11 @@ int CMPConnectThread::run()
 #if defined(_USE_OPENSSL)
             if (parent->useTLS)
             {
-                Owned<ISecureSocket> ssock = secureContextServer->createSecureSocket(sock.getClear());
                 int tlsTraceLevel = SSLogMin;
                 if (parent->mpTraceLevel >= MPVerboseMsgThreshold)
                     tlsTraceLevel = SSLogMax;
-                int status = ssock->secure_accept(tlsTraceLevel);
+                Owned<ISecureSocket> ssock = secureContextServer->createSecureSocket(sock.getClear(), tlsTraceLevel);
+                int status = ssock->secure_accept(10000);
```

Comment:
see other comment(s). There's a few different constants used for secure_connect/secure_accept, in most cases sensible defaults should be used. I think secure socket connect/accept should have different defaults to jsocket's raw socket connect/accept. They are not expected to have the same timing semantics.